features/step_definitions/epics_steps/epics_steps.rb

```diff
@@ -43,14 +45,17 @@
 
 When (/I sending a DELETE request to (.*) endpoint/) do |endpoint|
   # noinspection RubyResolve
-  @code = @epic.epics_delete(@client, endpoint, @project_id, @epic_id)
+  if @epic_id == nil
+    @epic_id = ''
+  end
+  @code, @body = @epic.epics_delete(@client, endpoint, @project_id, @epic_id)
   # noinspection RubyResolve
-  @client.delete_request('projects/' + @project_id)
+  _, _ = @client.delete_request('projects/' + @project_id)
```

Comment:
insted of use _, _ you only can call to the method:
@client.delete_request('projects/' + @project_id)